kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml

```diff
@@ -2088,6 +2092,12 @@ components:
         - OFFSET
         - TIMESTAMP
 
+    MessageFilterType:
+      type: string
+      enum:
+        - STRING_CONTAINS
+        - GROOVY_SCRIPT
```

Comment:
Just an idea but to name this GROOVY_EXPRESSION since to me it feels more like an expression then an actual script

Reply:
we support multiline, multistatement scripts, so I think it is legal to call it "script"